requirements.txt

```diff
@@ -1 +1,2 @@
-requests==2.9.1
\ No newline at end of file
+requests==2.20.1
+tox==3.5.3
```

Comment:
You can add tox as a dev/test requirement in the setup file the following way:
```
    extras_require={
        'dev': [
            'tox',
        ]
    },
```
Then you can say the following:
```
python setup.py .[dev]
```
And it will nicely install `tox` for the testing environment

Reply:
```pip install -e ".[dev]"```